questions/qa-personal-names.fr.html

```diff
@@ -338,7 +337,7 @@ <h2 id="encoding"><a href="#encoding">Incidence sur le support de caractères</a
 
 
 <section>
-  <h2 id="bytheway"><a href="#bytheway">By the way</a></h2>
+  <h2 id="bytheway"><a href="#bytheway">À propos</a></h2>
```

Comment:
```suggestion
  <h2 id="bytheway"><a href="#bytheway">Au fait</a></h2>
```

(“À propos” is ambiguous in Web content as it means “About” most of the time.)

Reply:
I understand you’re referring to _À propos_ being used in the context of navigation, in which case it can be rightly understood as _About (this website)_.  The context of the article gives the same signal about the current context which is the article, which should avoid any confusion in this specific case.

Deal? No deal? :)

Reply:
Not convinced still, but it was just a suggestion, feel free to ignore me, I won't make a fuss about it ;)

Reply:
Other articles are mainly using _À propos_ (also _En passant_, _Par ailleurs..._ and _By the way_) so I will keep it like now.

I might make a commit to homogenize on this one is merged. :}

Reply:
Actually, [some JS](https://github.com/arkhi/i18n-drafts/blob/gh-pages/javascript/boilerplate-text/boilerplate-fr.js#L90) is replacing all #bytheway with _En passant_… ;)